src/classes/ObjectClass.ts

```diff
@@ -3,7 +3,7 @@ import { IObject } from "../interfaces/IObject";
 class ObjectClass {
 
   getImage(object: IObject): string {
-      const imagePath = process.env.VUE_APP_BACKEND + '/images/characters/';
+      const imagePath = process.env.VUE_APP_BACKEND + '/images/characters/test/';
```

Comment:
Please revert the path. For now we're just going to display the images that contain text as otherwise the game can't be played...